schema_based_on_diagram.sql

```diff
@@ -0,0 +1,47 @@
+CREATE DATABASE vet_clinic_diagram;
+-- Create a table named invoice_items  with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+--unit_price        DECIMAL,
+ -- quantity          INT,
+  --total_price       DECIMAL,
+  --invoice_id        INT REFERENCES invoices(id),
+  --treatment_id      INT REFERENCES treatments(id),
+  --PRIMARY KEY(id, invoice_id, treatment_id)
```

Comment:
- [ ] Please make sure all FOREIGN KEYS have indexes created as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/231500720-7f059d06-f556-4f18-8cfd-2c26656be31c.png)

Reply:
Sure I will create the FK indexes. Thanks